cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -684,6 +694,60 @@ fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(
             ctx.emit(Inst::setcc(cc, dst));
         }
 
+        Opcode::Fcmp => {
+            let condcode = inst_fp_condcode(ctx.data(insn)).unwrap();
+
+            let lhs = input_to_reg_mem(ctx, inputs[0]);
+            let rhs = input_to_reg(ctx, inputs[1]);
+            let dst = output_to_reg(ctx, outputs[0]);
+
+            // Unordered is returned by setting ZF, PF, CF <- 111
+            // Greater than by ZF, PF, CF <- 000
+            // Less than by ZF, PF, CF <- 001
+            // Equal by ZF, PF, CF <- 100
+
+            // Because setcc check together the CF and ZF, we swap
+            // operands for condition codes that would require checking this
+            // flags in a single setcc operation, namley LessThan and LessThanOrEqual.
+            // Because setcc does not check together the PF with either ZF or CF
+            // we swap operands for condition codes that would benefit by , namely
```

Comment:
nit: missing word after `benefit by`? (or at least an empty space to remove)

Reply:
I've cleaned all this up.